crates/router/src/connector/fiserv/transformers.rs

```diff
@@ -114,9 +114,10 @@ impl TryFrom<&types::PaymentsAuthorizeRouterData> for FiservPaymentsRequest {
             reversal_reason_code: None,
         };
         let metadata = item.get_connector_meta()?;
-        let session: SessionObject = metadata
-            .parse_value("SessionObject")
-            .change_context(errors::ConnectorError::RequestEncodingFailed)?;
+        let session: FiservSessionObject =
+            metadata.parse_value("FiservSessionObject").change_context(
+                errors::ConnectorError::InvalidConnectorConfig { config: "metadata" },
```

Comment:
We have metadata in Payments request also, So here let's give the clarity of Merchant Connector account metadata. The same comment is applicable for all the places where ever you are using "metadata" for the deserialization error

Reply:
Done!